src/cli/index.ts

```diff
@@ -7,11 +7,6 @@ export function createStarbaseCli() {
   return createCommand()
     .name('yarn starbase')
     .description('Starbase graph ingestion orchestrator')
```

Comment:
Commander was upgraded several major revisions in the SDK.  This changes how the .addCommand function works, namely that it no longer passes down parent flags and options.  Moving this to the run and setup commands.

Reply:
Thanks for the explanation